### DISCUSSION TITLE
Let user to define custom routes, menus, components

### DISCUSSION THREAD

**Discussion by c6p** [ananthakumaran/paisa#189] [Ideas]
First, thank you for your work.

Paisa have a very strong foundation, however I have different UI needs. I think only a few of them aligns with Paisa, though it seems pull requests are not encouraged.

Let user to define custom routes, menus, components in user directory, so a power-user can create custom dashboards.

**Comment by ananthakumaran** [comment]
I don't think this fits with what I have in mind. Too complex to get into it. You can specify your use case and if it's common enough, I can think about adding it to paisa.
